src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -56,6 +56,7 @@ type TransactionRowAsset = {
   amount: string
   precision: number
   currentPrice?: string
+  icon?: string
```

Comment:
Do we have a fallback icon we could use and make this required?

Reply:
We have a fallback in the sense we show a default icon when no icon can be loaded.